src/libsystemd/sd-device/device-monitor.c

```diff
@@ -46,6 +47,9 @@ struct sd_device_monitor {
         union sockaddr_union snl_trusted_sender;
         bool bound;
 
+        UidRange *uid_range;
```

Comment:
maybe call this `mapped_userns_uid_range`, to clarify what kind of uid range this is